app/src/main/java/org/piramalswasthya/cho/ui/commons/fhir_visit_details/ChiefComplaintFragment.kt

```
@@ -92,14 +92,21 @@ class ChiefComplaintFragment(private var chiefComplaintList: List<ChiefComplaint
+        try {
+            val description = binding.descInputText.text.toString().trim()
+            val durationUnit = binding.dropdownDurUnit.text.toString().trim()
+            val duration = binding.duration.inputDuration.text.toString().trim()
+            val chiefComplaint = binding.chiefComplaintDropDowns.text.toString().trim()
+
+            binding.resetButton.isEnabled = description.isNotEmpty() ||
+                    durationUnit.isNotEmpty() ||
+                    duration.isNotEmpty() ||
+                    chiefComplaint.isNotEmpty()
+
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)